lib/recurly/pager.php

```diff
@@ -45,15 +44,14 @@ public function rewind() {
    */
   public function current()
   {
-    if (empty($this->_count)) {
+    if ($this->count() == 0) {
```

Comment:
Ah, now I remember on older versions of PHP you can only call `empty()` on a writable target, not the result of a function call:
```
PHP Fatal error:  Can't use method return value in write context in /home/travis/build/recurly/recurly-client-php/lib/recurly/pager.php on line 47
```
I'll back that part out tomorrow and add a comment.